### DISCUSSION TITLE
Offical ps_trackingfront module not functional

### DISCUSSION THREAD

**Discussion by dialt0n3** [PrestaShop/PrestaShop#33923] [Q&A] [answered]
When Accessing Tracking Front Stats URL http://xxxxxxxx.com/modules/trackingfront/stats.php

Error 500 is displayed

with debug on it shows this: 

(1/1) ContextErrorExceptionNotice: Undefined index: tpl_dir
--
in 3246e59a08f83a5b1f092fd46e09865ae45deef0_2.file.login.tpl.php line 33
at content_6500697d254fe1_21150146(object(SmartyCustomTemplate))in smarty_template_resource_base.php line 123
at Smarty_Template_Resource_Base->getRenderedTemplateCode(object(SmartyCustomTemplate))in smarty_template_compiled.php line 114
at Smarty_Template_Compiled->render(object(SmartyCustomTemplate))in smarty_internal_template.php line 216
at Smarty_Internal_Template->render(false, 0)in smarty_internal_templatebase.php line 232
at Smarty_Internal_TemplateBase->_execute(object(SmartyCustomTemplate), null, null, null, 0)in smarty_internal_templatebase.php line 116
at Smarty_Internal_TemplateBase->fetch(null, null, null, null, false, true, false)in SmartyCustomTemplate.php line 44
at SmartyCustomTemplateCore->fetch()in Module.php line 2439
at ModuleCore->display('/home/seveastg/public_html/modules/trackingfront/trackingfront.php', 'login.tpl')in trackingfront.php line 169
at TrackingFront->displayLogin()in stats.php line 35

**Comment by MatShir** [comment, marked as answer]
Hi @dialt0n3,
The module is archived and not maintained anymore. You will need to find another module or ask a professional to fix the bug. 
Best regards
